content/_changelogs/12.0.0.md

```diff
@@ -1,26 +1,197 @@
+  [#24976](https://github.com/cypress-io/cypress/issues/24976) for more
+  information. Addressed in
+  [#24931](https://github.com/cypress-io/cypress/pull/24931).
+- Added a new `Cypress.ensure` API which provides several methods that can be
+  helpful when writing customs commands. Addressed in
+  [#24697](https://github.com/cypress-io/cypress/pull/24697).
+- Added a new "Run All" button to the
+  [`experimentalRunAllSpecs`](guides/references/experiments#End-to-End-Testing)
+  experiment which enables running all spec files returned from the
+  `specPattern` glob or an array of globs, regardless of the specs sharing a
+  root folder. Addresses
+  [#24759](https://github.com/cypress-io/cypress/issues/24759).
+- Further improved bundling in the binary to reduce startup and unzip time.
+  Addressed in [#24909](https://github.com/cypress-io/cypress/pull/24909)
+
+**Bugfixes:**
+
+- Fixed a regression in [10.11.0](#10-11-0) where the Launchpad no longer showed
+  a loading spinner on startup to indicate that it had successfully started and
+  was in a loading state. Fixed
+  [#24950](https://github.com/cypress-io/cypress/issues/24950).
+- Fixed the "Create Spec from Component" capability to correctly scaffold the
+  component spec from a component which contains special characters in its name.
+  Before this change, a spec was scaffolded with either the wrong spec name or
+  the incorrect component import which lead to invalid JS errors. Fixes
+  [#23492](https://github.com/cypress-io/cypress/issues/23492).
+- Corrected Cypress environment variable resolution to correctly resolve
+  environment variables set with `npm config set`. Fixes
+  [#24556](https://github.com/cypress-io/cypress/issues/24556).
+- When
+  [`experimentalRunAllSpecs=true`](guides/references/experiments#End-to-End-Testing),
+  the "Run N specs" directory-level buttons in the Inline Specs List now
+  function correctly when activated by the keyboard. Fixes
+  [#24762](https://github.com/cypress-io/cypress/pull/24762).
+- Fixed unexpected behaviors when verifying a cookie was not returned from not
+  returned from [`cy.getCookie()`](/api/commands/getcookie). Now it's possible
+  to explicitly assert the cookie did not exist with
+  `cy.getCookie('foo1').should('not.exist')`. Previously users had to verify
+  cookies properties did not exist with `.its('value').should('equal', null)`.
+  Addressed in [#24203](https://github.com/cypress-io/cypress/pull/24203).
+- Fixed the console output returned from clicking on an aliased intercept in the
+  Command Log. It now logs the yielded request/response object associated to the
+  intercepted call when clicked, where previously it did not log these details.
+  Fixed in [#24623](https://github.com/cypress-io/cypress/pull/24623).
+- Added missing Typescript types for the internal
+  [`cy.now()`](/guides/guides/debugging#Run-Cypress-command-outside-the-test)
+  command. Fixed in [#24697](https://github.com/cypress-io/cypress/pull/24697).
```

Comment:
Seems odd to put changes to an internal command in the changelog.

Reply:
I'm totally fine taking this out. I wasn't too sure if we should add or not.